src/autogpt_plugins/SocialMediaManager/__init__.py

```diff
@@ -0,0 +1,384 @@
+from typing import Any, Dict, List, Optional, Tuple, TypedDict, TypeVar
+from auto_gpt_plugin_template import AutoGPTPluginTemplate
+from .Instagram_Handler import load_ig_credentials, connect_ig, ig_post_photo, ig_post_video, ig_get_user_profile, ig_get_media_insights, ig_search_user, ig_get_followers, ig_get_following, ig_get_user_posts, ig_send_direct_message, ig_get_direct_messages, ig_get_hashtag_posts, ig_get_user_stories, fb_get_ig_post_insights
+from .Facebook_Handler import load_fb_credentials, connect_fb, fb_post_status, fb_post_photo, fb_post_video, fb_get_user_profile, fb_get_page_insights, fb_search_pages, fb_get_page_posts, fb_get_post_comments, fb_get_page_reviews, fb_get_page_events, fb_create_page_event, fb_get_group_members, fb_get_group_posts
+from .Twitter_Handler import load_tw_credentials, connect_tw, tw_get_followers, tw_post_tweet, tw_post_media_tweet, tw_get_user_profile, tw_get_user_insights, tw_search_users, tw_get_followers, tw_get_following, tw_get_user_tweets, tw_get_tweet_replies, tw_get_trending_topics, tw_search_tweets, tw_get_hashtag_tweets, tw_get_tweet_insights
+from .Google_My_Business_Handler import load_gmb_credentials, connect_gmb, gmb_create_post, gmb_create_event, gmb_get_location_insights, gmb_get_location_reviews, gmb_get_location_posts, gmb_get_location_photos, gmb_get_location_videos, gmb_upload_photo, gmb_upload_video, gmb_get_review_replies, gmb_reply_to_review, gmb_update_location_info, gmb_get_location_metadata
+
+PromptGenerator = TypeVar("PromptGenerator")
+
+class Message(TypedDict):
+    role: str
+    content: str
+
+class AutoGPTSocialMediaManager(AutoGPTPluginTemplate):
+    """
+    Social Media Manager plugin for Instagram, Facebook, Twitter, and Google My Business Profile
+    """
+
+    def __init__(self):
+        super().__init__()
+        self._name = "AutoGPT-Social-Media-Manager"
+        self._version = "1.0.0"
+        self._description = "Social Media Manager plugin for Instagram, Facebook, Twitter, and Google My Business Profile. Created by https://wellnesssupply.co"
+        
+        load_ig_credentials()
+        load_fb_credentials()
+        load_tw_credentials()
+        load_gmb_credentials()
+
+        connect_ig()
+        connect_fb()
+        connect_tw()
+        connect_gmb()
```

Comment:
make all of these Social Media platforms classes so they aren't polluting the global context (and later on we can run multiple social accounts by loading the plugin with different configs)

Reply:
Appreciate it! Still learning. Will have an update later after work.